queue_services/entity-filer/k8s/templates/entity-filer-deploy.json

```diff
@@ -536,6 +536,13 @@
       "description": "Service Account timeout.",
       "required": true,
       "value": 20
+    },
+    {
+      "name": "NAMEX_API",
```

Comment:
need to add this in the config map and the deployment objects above too!

Reply:
doh